filter/etc/e-smith/templates/etc/rspamd/local.d/antivirus.conf/10base

```diff
@@ -33,8 +33,8 @@ clamav \{
 
   # Timeout and retransmits increased in case of clamav is reloading its database
   # It takes a lot of time (25 to 60 seconds), after rspamd answers a temporally failure
-  timeout = 45;
-  retransmits = 1;
+  timeout = $scanTimeout.0;
+  retransmits = 0;
```

Comment:
I trust rspamd making retries for a good reason. We always had at least one retransmit, I don't want to risk unexpected regressions removing them.

Reply:
lets go to 45 * 2 (one retransmit), I added some comment to avoid issues, because I  asked myself why rspamd has been wait 90 seconds with a prop of 45